autoscheduler/frontend/src/tests/redux/Schedule.test.tsx

```diff
@@ -0,0 +1,228 @@
+import { createStore } from 'redux';
```

Comment:
We should also probably have some basic UI tests for the entire Schedule page, i.e. clicking on a schedule makes the respective section(s) show up on the Schedule, etc